verify_rpms/rpm_verifier.py

```diff
@@ -112,16 +127,32 @@ def generate_output(processed_image: ProcessedImage) -> tuple[bool, str]:
     return failure, output.lstrip()
 
 
+def generate_results(processed_image: ProcessedImage) -> dict[str, Any]:
```

Comment:
I'd prefer not to have `Any` there. Maybe can do something like this:
```suggestion
def generate_results(processed_image: ProcessedImage) -> dict[str, Union[dict[str, str], str]]:
```

Reply:
Done